.circleci/config.yml

```diff
@@ -82,25 +82,19 @@ workflows:
           name: black
           command-run: |
             pip install --progress-bar off -e ".[dev-core]" && \
-            black --check src tools tests docs/source/conf.py
+            black --check src tools tests docs/conf.py
       - pre-build:
           name: flake8
           command-run: |
             pip install --progress-bar off -e ".[dev-core]" && \
-            flake8 src tools tests docs/source/conf.py
+            flake8 src tools tests docs/conf.py
       - pre-build:
           name: pip-audit
           command-run: |
             pip install --progress-bar off -r docs/requirements.txt && \
             pip install --progress-bar off -e ".[dev-core,dev-civisml]" && \
             pip-audit --version && \
             pip-audit --skip-editable
-      - pre-build:
-          name: sphinx-build
-          command-run: |
-            pip install --progress-bar off -r docs/requirements.txt && \
-            pip install --progress-bar off --no-deps -e . && \
-            sphinx-build -b html -n docs/source/ docs/build/
```

Comment:
Removing this sphinx-build step in CircleCI builds, because we now have a Read The Docs build whenever a pull request is created. Speaking of which, here's the doc build from this PR for our preview: https://civis-python--506.org.readthedocs.build/en/506/